Assets/Scripts/MovingEnemyCon.cs

```diff
@@ -9,16 +9,28 @@ public class MovingEnemyCon : MonoBehaviour
     private float curr;
     public float left_dist = 1;
     public float right_dist = 2;
+    public bool inverse;
     void Start()
     {
-        check = true;
+        inverse = false;
```

Comment:
Why this?
This essentially changes the code to -
```cs
if (false) {
    check = true;
}
else {
    check = false;
}
```

which is basically - 
```cs
check = false;
```

So why can't we just do `check = false;` instead ?